doc/ipythonconsole.rst

```diff
@@ -44,12 +44,7 @@ To connect to an external kernel,
 
 #. Copy the connection file (:file:`{runtime-dir}/kernel-{pid}.json`) to the machine you're running Spyder on (if remote) or note its location (if local).
 
-   You can get the location of :file:`{runtime-dir}` by starting an IPython interpreter in the same Python environment as the kernel and running:
-
-   .. code-block:: python
-
-      import jupyter_core
-      jupyter_core.paths.jupyter_runtime_dir()
+   You can get the :file:`{runtime-dir}` path by executing ``jupyter --runtime-dir`` in the same host environment as the kernel.
```

Comment:
host? Why not Python environment, as before?

Reply:
I tweaked it since the command is run in the shell, not in Python directly (obviously, it implicitly depends on the Python interpreter/environment path, but that isn't directly exposed to the user through this method), so I didn't want to confuse users into thinking the command needed to be run in Python. I can change it back or to something else; whatever you think is best.

Reply:
Well, if jupyter is not installed globally, this command is not going to work.

Reply:
So, the implication is you need to have the appropriate Anaconda/virtualenv environment, or other Python install activated or otherwise on your path? Or otherwise, how would the requirements for this command working differ from that for ``python -m spyder_kernels.console``? I'm not certain I follow, sorry.

Reply:
> Or otherwise, how would the requirements for this command working differ from that for python -m spyder_kernels.console?

When you install spyder-kernels you get the `jupyter` command line, so the requirements are exactly the same.

But you need to have Anaconda in your PATH or have your environment activated (if using venvs) to be able to use `jupyter`.

The only thing I find easier about this command is that you don't need to fire up an IPython console, `import jupyter_core` there, etc.

Reply:
> Of course, the same is true for python -m spyder_kernels.console

Yep, that's correct.

> I changed the wording back to "Python environment" instead of "host environment". If you want something additional or different, just let me know and I'll do it.

Thanks. That's all that I wanted.